components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -148,11 +149,9 @@ type GetTokenBalancesForChainIdArg =
   | GetSPLTokenBalancesForAddressAndChainIdArg
   | GetTokenBalancesForAddressAndChainIdArg
 
-type NetworkInfo = Pick<BraveWallet.NetworkInfo, 'chainId' | 'coin'>
-
 type GetTokenBalancesRegistryArg = {
   accountIds: BraveWallet.AccountId[]
-  networks: NetworkInfo[]
+  networks: BraveWallet.NetworkInfo[]
```

Comment:
How about this?
Will this work?
```suggestion
  networks: Pick<BraveWallet.NetworkInfo, 'chainId' | 'coin' | 'supportedKeyrings'>
```

Reply:
https://github.com/brave/brave-core/pull/20812/commits/419eda221b83c47e8041e3eb0ef9d94a46c772e5